src/EditorFeatures/Core/EmbeddedLanguages/AbstractEmbeddedLanguageEditorFeaturesProvider.cs

```diff
@@ -24,8 +24,7 @@ protected AbstractEmbeddedLanguageEditorFeaturesProvider(EmbeddedLanguageInfo in
             Languages = ImmutableArray.Create<IEmbeddedLanguage>(
                 new DateAndTimeEmbeddedLanguageEditorFeatures(info),
                 new JsonEmbeddedLanguageEditorFeatures(info),
-                new RegexEmbeddedLanguageEditorFeatures(this, info),
-                new FallbackEmbeddedLanguage(info));
+                new RegexEmbeddedLanguageEditorFeatures(this, info));
```

Comment:
intentional.  the fallbackembeddedlanguage went away.  it was only there for classification, and now classification is handled using a normal MEF registration path.